### DISCUSSION TITLE
Welcome to AMS Discussions!

### DISCUSSION THREAD

**Discussion by jinningwang** [CURENT/ams#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

**Comment by ZhihangMao** [comment]
When I run "import ams" in python, the following error message occurred:
```
Your generated pycode is broken. Run `andes prep` to re-generate. 
Generating code for 97 models on 24 processes.
Your generated pycode is broken. Run `andes prep` to re-generate. 
Generating code for 97 models on 24 processes.
Traceback (most recent call last):
......
RuntimeError: 
        An attempt has been made to start a new process before the
        current process has finished its bootstrapping phase.

        This probably means that you are not using fork to start your
        child processes and you have forgotten to use the proper idiom
        in the main module:

            if __name__ == '__main__':
                freeze_support()
                ...

        The "freeze_support()" line can be omitted if the program
        is not going to be frozen to produce an executable.

        To fix this issue, refer to the "Safe importing of main module"
        section in https://docs.python.org/3/library/multiprocessing.html
        
Your generated pycode is broken. Run `andes prep` to re-generate. 
```
I tried both the pip and conda installation methods in the virtual environment according to the README.md, but neither could solve the problem. Is there any solution? Thanks!

**Reply by jinningwang** [reply]
Check this, https://github.com/CURENT/andes/issues/508

**Reply by jinningwang** [reply]
@ZhihangMao Hi, does this address your concern? If not feel free to open an issue where you can give more details.
